front/reducers/itinerary-reducer.js

```diff
@@ -0,0 +1,20 @@
+const defaultState = {
+  wanderSpots: []
+};
+
+const itineraryReducer = (state=defaultState, action) => {
+  switch(action.type) {
+    case 'GET_WANDERSPOTS': {
+      return {...state}
+      break;
+    }
+    case 'ADD_MARKERS_FULFILLED': {
+      //NEED HELP WITH THE RETURN... action.getWanderspot
+      return {...state, wanderSpots: state.wanderSpots}
```

Comment:
since the idea is to keep the state/store immutable, and the default state has an empty array to begin with, you might want to concat your data to wanderSpots.
It should look something like this:
{...state, wanderSpots: state.wanderSpots.concat(action.payload)}

We are doing this because action.payload, which is the data we receive from our API get request is returning an array of object.